$NetBSD: patch-aa,v 1.12 2015/07/03 12:37:44 wiz Exp $

Compile 'strip' test with CFLAGS, so -isysroot can be passed on OSX.
Correct library detection on NetBSD.

--- config.lib.orig	2015-06-01 18:43:32.000000000 +0000
+++ config.lib
@@ -2070,7 +2070,7 @@ check_strip() {
 	elif [ "$os" = "OSX" ]; then
 		# Most targets have -V in strip, to see if they exists... OSX doesn't.. so execute something
 		echo "int main(int argc, char *argv[]) { }" > strip.test.c
-		$cxx_host strip.test.c -o strip.test
+		$cc_host $CFLAGS strip.test.c -o strip.test
 		check_compiler "host strip" "strip" "$host" "$strip" "$STRIP" "strip" "strip" "3" "strip.test"
 		rm -f strip.test.c strip.test
 	else
@@ -2628,7 +2628,7 @@ detect_library() {
 		if [ -z "$res" ]; then
 			log 2 "  trying /opt/local/include/$4$5... no"
 		fi
-		if [ -z "$res" ] && [ "$os" = "NETBSD" ]; then
+		if [ -z "$res" ]; then
 			eval "$2=`ls -1 /usr/pkg/include/$4*.h 2>/dev/null | egrep \"\/$5\$\"`"
 			eval "res=\$$2"
 			if [ -z "$res" ]; then
